src/Internal.AspNetCore.HostingStartup.Sdk/build/Internal.AspNetCore.HostingStartup.Sdk.targets

```diff
@@ -0,0 +1,50 @@
+<!--
+  This file is automatically imported by NuGet into a user's project
+  when it targets a single framework, or in classic (pre 2017) csproj projects.
+-->
+
+<Project>
+
+  <Target Name="GenerateHostingStartupDeps" BeforeTargets="Build" Condition="'@(HostingStartupPackageReference->Count())' != '0'">
+    <PropertyGroup>
+      <_TemplatesDirectory>$(MSBuildThisFileDirectory)..\content\</_TemplatesDirectory>
+      <_DepsOutputDirectory>$(MSBuildProjectDirectory)\$(BaseIntermediateOutputPath)</_DepsOutputDirectory>
```

Comment:
Use `$(IntermediateOutputPath)` instead of BaseIntermediateOutputPath. This will already have been normalized for config + target framework, and have been turned into a full path.